### PR TITLE
Add notes about brotli in the browser

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: aspnetcore/host-and-deploy/blazor/webassembly.md [hunk 1]
```diff
@@ -23,13 +23,43 @@ The following deployment strategies are supported:
 * The Blazor app is served by an ASP.NET Core app. This strategy is covered in the [Hosted deployment with ASP.NET Core](#hosted-deployment-with-aspnet-core) section.
 * The Blazor app is placed on a static hosting web server or service, where .NET isn't used to serve the Blazor app. This strategy is covered in the [Standalone deployment](#standalone-deployment) section, which includes information on hosting a Blazor WebAssembly app as an IIS sub-app.
 
-## Precompression
+## Compression
```
Comment (pranavkm) [inline aspnetcore/host-and-deploy/blazor/webassembly.md:26]: *Pre* sounds weird.

File: aspnetcore/host-and-deploy/blazor/webassembly.md [hunk 2]
```diff
@@ -23,13 +23,43 @@ The following deployment strategies are supported:
...
+When a Blazor WebAssembly app is published, the output is statically compressed during publish to reduce the app's size and remove the overhead for runtime compression. The following compression algorithms are used:
 
 * [Brotli](https://tools.ietf.org/html/rfc7932) (highest level)
 * [Gzip](https://tools.ietf.org/html/rfc1952)
 
+Blazor's relies on the host to the server the appropriate compressed file. When using an ASP.NET Core hosted project, the host project is capable of performing content negotiation and serving the statically compressed file. When hosting a Blazor WebAssembly self-hosted application, you may have to do additional work to ensure statically compressed are served.
+
+* For IIS *web.config* compression configuration, see the [IIS: Brotli and Gzip compression](#brotli-and-gzip-compression) section. 
+* When hosting on static hosting solutions (such as GitHub Pages) that do not support this form of content negotiation, consider configuring your application to fetch and decode Brotli compressed files:
```
Comment (pranavkm) [inline aspnetcore/host-and-deploy/blazor/webassembly.md:36]: It might be useful to point to this from the perf docs

Comment (guardrex) [reply aspnetcore/host-and-deploy/blazor/webassembly.md:36]: Do you want to add it over there to this PR, or do you want me to do it later?

The cross-link markdown is ...

```
<xref:host-and-deploy/blazor/webassembly#compression>
```

Comment (pranavkm) [reply aspnetcore/host-and-deploy/blazor/webassembly.md]: I can do it in a follow up.

Comment (guardrex) [reply aspnetcore/host-and-deploy/blazor/webassembly.md]: Ok, I'll hold off on the merge.

Comment (guardrex) [reply aspnetcore/host-and-deploy/blazor/webassembly.md]: > follow up

Follow-up *commit* or follow-up *PR*?

**Comment by pranavkm** [comment]
Let's get this one in, I'll send a follow up PR

**Pull request merged**